jenkins/ci.suse.de/openstack-ardana.yaml

```diff
@@ -164,8 +181,25 @@
                   ;;
           esac
 
+          # Simple build pool management
+          if [ -n "$build_pool_name" ]; then
```

Comment:
For consistency and also to reduce external dependencies, I would prefer `[[` instead of `[`.